ambari-project/pom.xml

```diff
@@ -512,17 +511,22 @@
       <dependency>
         <groupId>com.fasterxml.jackson.core</groupId>
         <artifactId>jackson-annotations</artifactId>
-        <version>${fasterxml.jackson.version}</version>
+        <version>2.9.8</version>
       </dependency>
```

Comment:
Why not change ${fasterxml.jackson.version} to 2.9.8?

Reply:
as i have moved from setting variable everywhere to using dependencyManagement plugin